common/scripts/fix-version.js

```diff
@@ -0,0 +1,16 @@
+// Helper script used to fix package.json version resetting on builds, and helping bump version.json after releases
+
+// Copies the version from version.json into package.json
+var fs = require("fs");
+var path = require("path");
+
+var folder = process.argv[2];
```

Comment:
could we add a check for the length of this array?
maybe return exit 1 (to make the release fail in case of incorrect args?)

Reply:
address in another PR